docs/cookbook/admin_panel/menu.md

```diff
@@ -88,3 +88,45 @@ final readonly class MenuBuilder implements MenuBuilderInterface
     }
 }
 ```
+
+### Opening a link in a new tab
+
+Target blank directive can be added to link that either are top level menu items without children or children items. In other words this can be configured to item that do not have children.
+
+This can be configured by setting a link attribute on a menu item:
+
+You can make a link open in a new browser tab by adding the `target="_blank"` link attribute to menu items. This can be applied to top-level menu items without children, or to child items themselves. In other words, it only works for items that do not have submenus.
+
+To configure this behavior, set the target link attribute on the menu item as shown below:
+
+```php
+// ...
+#[AsDecorator(decorates: 'sylius_admin_ui.knp.menu_builder')]
+final readonly class MenuBuilder implements MenuBuilderInterface
+{
+    // ...
+
+    public function createMenu(array $options): ItemInterface
+    {
+        $menu = $this->factory->createItem('root');
+        // ...
+        $this->aaddApiDocMenuddApiDocMenu($menu);
+
+        return $menu;
+    }
+
+    private function addApiDocMenu(ItemInterface $menu): void
```

Comment:
Personally, I would prefer some other example external submenu/url name than api doc, something in line with books and library from test app and other example snippets.

WDYT @loic425 @stlgaits

Reply:
Yes good point to try to remain in userland - possibly an external link to the bookstore's website / blog or to something related to the books ecosystem like Goodreads  new releases  for instance (e.g. https://www.goodreads.com/book/popular_by_date/2025/5?ref=nav_brws_newrels) ?

Reply:
I changed it to a link to the website homepage. This is not in the test app, but we've got something similar in `Sylius/Sylius`